wgpu/backends/rs.py

```diff
@@ -339,7 +347,11 @@ def to_py_str(key):
         # WebGPU features
         features = set()
         for f in sorted(enums.FeatureName):
-            i = enummap[f"FeatureName.{f}"]
+            key = f"FeatureName.{f}"
+            if key not in enummap:
+                assert f in KNOWN_MISSING_FEATURES
```

Comment:
I strongly advise against using `assert` (especially without a message) in user code (anything outside a `tests` folder)

Reply:
Fixed!